src/BizHawk.Tests/Client.Common/Movie/ZwinderStateManagerTests.cs

```diff
@@ -413,7 +413,7 @@ public void StateCache()
 
 			for (int i = 400; i < 1000; i += 400)
 			{
-				zw.EvictReserved(i);
+				zw.Unreserve(i);

```

Comment:
Would this pass if it was still calling `EvictReserved`?

Reply:
Aside from the fact that it would no longer compile (since EvictReserved is and should be private (why doesn't C# let unit tests call privates?)), yes it would.